gui/launcher.lua

```diff
@@ -258,6 +269,34 @@ function EditPanel:init()
             on_submit2=function()
                 self.on_submit2(self.subviews.editfield.text) end},
     }
+    self:update_pause_label(false,true)
+end
+
+function EditPanel:pause_fort()
+    if dfhack.isMapLoaded() then
+        df.global.pause_state = not df.global.pause_state
+    end
+    self:update_pause_label(false)
+end
+function EditPanel:enable_pause_label()
+    self.subviews.pauselabel.text[1].pen = COLOR_WHITE
+    self.subviews.pauselabel.text[1].key_pen = COLOR_LIGHTGREEN
+end
+function EditPanel:disable_pause_label()
+    self.subviews.pauselabel.text[1].pen = COLOR_DARKGREY
+    self.subviews.pauselabel.text[1].key_pen = COLOR_GREEN
+end
+function EditPanel:update_pause_label(willchange, canchange)
+    if dfhack.isMapLoaded() then
+        if canchange ~= nil then
+            if canchange then self:enable_pause_label()
+            else self:disable_pause_label() end
+        end
+        self.subviews.pauselabel.text[1].text = (df.global.pause_state ~= willchange) and 'unpause' or 'pause'
+    else
+        self:disable_pause_label()
+        self.subviews.pauselabel.text[1].text = 'no fort loaded'
```

Comment:
You should probably disable the Label in this case so it doesn't react on hover.
![image](https://user-images.githubusercontent.com/977482/229654443-d0dc30f8-a869-4ae4-b140-42bc2d33a640.png)